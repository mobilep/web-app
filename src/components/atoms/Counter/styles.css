.Counter {
	font-size: 12px;
	color: var(--almostBlack);
}
