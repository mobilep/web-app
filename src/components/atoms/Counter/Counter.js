import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';

import './styles.css';

export default class Counter extends PureComponent {

	static propTypes = {
		children: PropTypes.number,
		className: PropTypes.string,
	}

	getClassName = (className) => {
		return classNames({
			Counter: true,
		}, className);
	}

	render () {
		const className = this.getClassName(this.props.className);

		return <div className={className}>{this.props.children}</div>;
	}

}
