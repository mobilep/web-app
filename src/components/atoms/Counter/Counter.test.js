import React from 'react';
import { shallow } from 'enzyme';

import Counter from './Counter';

describe('Atom - Counter', () => {

	it('Should render', () => {
		const wrapper = shallow(
			<Counter>
				{15}
			</Counter>
		);
		expect(wrapper).toMatchSnapshot();
	});

});
