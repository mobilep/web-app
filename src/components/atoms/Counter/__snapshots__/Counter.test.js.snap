// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom - Counter Should render 1`] = `
<div
  className="Counter"
>
  15
</div>
`;
